tests/integration/test_convolve.py

```diff
@@ -25,6 +25,29 @@
 FILTER_SHAPES = [(5,), (3, 5), (3, 5, 3)]
 
 
+@pytest.mark.xfail
+def test_none():
+    # Numpy raises:
+    # TypeError: unsupported operand type(s) for *: 'NoneType' and 'NoneType'
+    with pytest.raises(AttributeError):
+        num.convolve(None, None, mode="same")
+
+
+def test_empty():
+    msg = r"empty"
+    with pytest.raises(ValueError, match=msg):
```

Comment:
@manopapad do you think matching partial exception messages (optionally) is OK? Or do we strictly only want to check type going forward?

Reply:
This level of detail makes sense to me. You'd hope an error message about an empty array would contain the word "empty".